website/docs/r/nginx_deployment.html.markdown

```diff
@@ -170,6 +170,13 @@ An `auto_scale_profile` block supports the following:
 
 -> **NOTE:** If you're using autoscaling with deployments created before v4.0, you may need to use [Terraform's `ignore_changes` functionality](https://www.terraform.io/language/meta-arguments/lifecycle#ignore_changes) to ignore changes to the `capacity` field.
 
+A `nginx_app_protect` block exports the following:
```

Comment:
This is resource. I think you want to say that the block `supports` the following.